generators/client/templates/react/package.json.ejs

```diff
@@ -173,7 +173,7 @@ limitations under the License.
     "xml2js": "0.4.19"<% } %>
   },
   "resolutions": {
-    "@types/react": "16.8.4"
+    "@types/react": "16.8.10"
```

Comment:
Does it work without the resolution?  That was originally added a long time ago so it might no longer be needed

Reply:
Let me test it

Reply:
No, apparently not for npm at least. Here's part of the build log:
```
[INFO] ERROR in /home/pav/Development/temp/jhipster-java/react-default/src/test/javascript/spec/app/shared/layout/header/menus/account.spec.tsx
[INFO] ERROR in /home/pav/Development/temp/jhipster-java/react-default/src/test/javascript/spec/app/shared/layout/header/menus/account.spec.tsx(4,10):
[INFO] TS2305: Module '"../../../../../../../../main/webapp/app/shared/layout/header/header-components"' has no exported member 'NavDropdown'.
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] npm ERR! errno 2
[ERROR] npm ERR! travis-react@0.0.0 webpack: `node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js "--config" "webpack/webpack.dev.js" "--env.stats=minimal"`
[ERROR] npm ERR! Exit status 2
[ERROR] npm ERR! 
[ERROR] npm ERR! Failed at the travis-react@0.0.0 webpack script.
[ERROR] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[ERROR] 
[ERROR] npm ERR! A complete log of this run can be found in:
[ERROR] npm ERR!     /home/pav/.npm/_logs/2019-03-29T18_50_13_365Z-debug.log
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] npm ERR! errno 2
[ERROR] npm ERR! travis-react@0.0.0 webpack:build:main: `npm run webpack -- --config webpack/webpack.dev.js --env.stats=minimal`
[ERROR] npm ERR! Exit status 2
[ERROR] npm ERR! 
[ERROR] npm ERR! Failed at the travis-react@0.0.0 webpack:build:main script.
[ERROR] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[ERROR] 
[ERROR] npm ERR! A complete log of this run can be found in:
[ERROR] npm ERR!     /home/pav/.npm/_logs/2019-03-29T18_50_13_557Z-debug.log
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] npm ERR! errno 2
[ERROR] npm ERR! travis-react@0.0.0 webpack:build: `npm run cleanup && npm run webpack:build:main`
[ERROR] npm ERR! Exit status 2
[ERROR] npm ERR! 
[ERROR] npm ERR! Failed at the travis-react@0.0.0 webpack:build script.
[ERROR] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[ERROR] 
[ERROR] npm ERR! A complete log of this run can be found in:
[ERROR] npm ERR!     /home/pav/.npm/_logs/2019-03-29T18_50_13_763Z-debug.log

```

Reply:
It works with yarn though. Got to :heart: frontend dev...

Reply:
I think above errors are related to recent changes to move menu components and should appear during upgrade flow. Can you assert with fresh project?

Reply:
@vishal423 You are right. Did a clean build and it works with npm as well. Will remove resolutions then and wait for CI. Thanks

Reply:
yes, i have raised separate PR to cleanup those files during upgrade. It would be really good to have test scenarios that does upgrade to catch such issues (daily builds should be good enough).

Reply:
Yes, it would be nice but also a bit difficult to implement. PRs are always welcome :)